xc/domain.ml

```diff
@@ -706,6 +706,7 @@ let xenguest_args_hvm ~domid ~store_port ~store_domid ~console_port ~console_dom
   ]
   @ (vgpus |> function
     | [Xenops_interface.Vgpu.{implementation = Nvidia _}] -> ["-vgpu"]
+    | Xenops_interface.Vgpu.{implementation = Nvidia _} :: _ -> ["-vgpu"]
```

Comment:
Do we need a Unit Test for the the xenguest also? so that similar argument will be tested and found during Unit Test phase.